File: rt/emdata/ccf2.cpp
#include "emdata.h"
#include "testutil.h"

using namespace EMAN;

int main()
{
	EMData *a = new EMData();
	a->read_image(TestUtil::get_debug_image("3d86_1.mrc"));

	EMData *b = new EMData();
	b->read_image(TestUtil::get_debug_image("3d86_2.mrc"));
#if 1
	EMData *c = a->calc_ccf(b, 0);
	delete b;
	b = 0;
#endif
	
#if 0
	EMData *fft = a->do_fft();
	EMData *c = fft->copy(false);
#endif
	
	c->write_image("aatest2.mrc");

	delete a;
	a = 0;

	
	return 0;
}

File: rt/pyem/ccf2.py
#!/bin/env python

from EMAN2 import *


img1 = Util.get_debug_image("3d86_1.mrc")
img2 = Util.get_debug_image("3d86_2.mrc")

a=EMData()
b=EMData()
a.read_image(img1)
b.read_image(img2)
c=a.calc_ccf(b,1)
c.write_image("test_ccf2.mrc")
